plugins/emoji/plugin.js

```diff
@@ -629,6 +629,16 @@
 						data = arrTools.filter( emojiList, function( item ) {
 							// Comparing lowercased strings, because emoji should be case insensitive (#2167).
 							return item.id.toLowerCase().indexOf( emojiName ) !== -1;
+						} ).sort( function( a, b ) {
+							var aIndex = a.id.substr( 1 ).indexOf( emojiName ),
+								bIndex = b.id.substr( 1 ).indexOf( emojiName );
+
+							// True when left and right side are oposite.
```

Comment:
```suggestion
							// True when left and right side are opposite.
```

Reply:
Still this comment will not be precise. This condition is only true if *only one* of sorted items begins with emojiName.

I'd understand: "left and right side are opposite" like if one has the match at the beginning, and the other at the end.